mongo/client_examples_test.go

```diff
@@ -468,3 +469,252 @@ func ExampleConnect_bSONOptions() {
 		panic(err)
 	}
 }
+
+func ExampleConnect_oIDC() {
```

Comment:
```suggestion
func ExampleConnect_OIDC() {
```

Reply:
Ah, this is something the other examples were doing and I thought it was necessary to suppress underscore warnings or something.